content/en/notebooks/guide/build_diagrams_with_mermaidjs.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Build diagrams with Mermaid JS
+kind: guide
+disable_toc: false
+---
+
+## Overview
+
+Datadog Notebooks supports Mermaid JS in any Markdown cell. 
+Mermaid is a compact notation for creating diagrams and flowcharts in Markdown. Learn more about Mermaid in the [official documentation][1] or by using the [Mermaid live editor][2].
+
+## Setup
+
+To create a new diagram in your notebook:
+1. Add a new text cell.
+1. Select **Code Block** from inside the Markdown editor.
+1. Select **Mermaid-JS**.
+
+In the Datadog site, there is also a [Build Diagrams with Mermaid JS][3] template, which you can use to get started and to see diagram examples.
+
+Use the [mermaid syntax][4] to build flowcharts, sequence diagrams, user journeys, gantt charts, and more.
+
+## Example
+
+Build a basic flowchart with a top down orientation:
+
+{{< code-block lang="shell" disable_copy="true" >}}
+```mermaid
+graph TD;
+    A-->B;
+    A-->C;
+    B-->D;
+    C-->E;
+```
+{{< /code-block >}}
+
```

Comment:
I would recommend adding a short sentence or description here of what the screencapture shows.